test/to_string_test.hpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+using namespace std;
+void toStringTest()
+{
+    int i;
+    cin>>i;
+    string  palindrom = to_string (i);
+    cout<<palindrom<<" testl"<<endl;
+}
```

Comment:
New line at the end of file is more friendly for `git diff` if someone add something behind.
```suggestion
}

```